Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nginx
```

Comment:
Oh, but then you'll need to project checked out. No one wants to check this project out every time. Lets build it with oculus/nginx, but try and keep our proprietary stuff out since this is still a public repo.